From baf6de35d80d4f8ef374050148c0b250cf62ceeb Fri, 23 Feb 2018 17:06:47 +0800
From: Heidi <Heidi@heidis-macbook-pro.local>
Date: Fri, 23 Feb 2018 16:48:43 +0800
Subject: [PATCH] Implemented the min/max functionalities

diff --git a/src/labs/comp3111/cse/ust/hk/Lab1Activity.java b/src/labs/comp3111/cse/ust/hk/Lab1Activity.java
index 0bba6bc..893b98e 100644
--- a/src/labs/comp3111/cse/ust/hk/Lab1Activity.java
+++ b/src/labs/comp3111/cse/ust/hk/Lab1Activity.java
@@ -6,11 +6,25 @@
 		int sum = 0;
  
 		//write your code here
-		for(int i= 0; i<= 10; i++){
-			sum += i;
+		for(int i= 0; i< 10; i++){
+			sum += arr[i];
+		}
+		
+		int min = arr[0];
+		for (int i = 0; i< 10; i++) {
+			if (arr[i] < min)
+				min = arr[i];
+		}
+		
+		int max = arr[0];
+		for (int i = 0; i< 10; i++){
+			if (arr[i] > max)
+				max = arr[i];
 		}
 		
 		System.out.println(String.format("Sum of the number is %d",sum));
+		System.out.println(String.format("Min = %d", min) + "; " + "Max = " + max + ";");
+		
 	}
 
 }